pkg/controller/sparkapplication/controller.go

```diff
@@ -292,19 +297,7 @@ func (c *Controller) getAndUpdateAppState(app *v1beta1.SparkApplication) error {
 				currentDriverState.completionTime = metav1.Now()
 			}
 		}
-		if util.IsExecutorPod(pod) {
-			newState := podPhaseToExecutorState(pod.Status.Phase)
-			oldState, exists := app.Status.ExecutorState[pod.Name]
-			// Only record an executor event if the executor state is new or it has changed.
-			if !exists || newState != oldState {
-				c.recordExecutorEvent(app, newState, pod.Name)
-			}
-			executorStateMap[pod.Name] = newState
-
-			if executorApplicationID == "" {
-				executorApplicationID = getSparkApplicationID(pod)
-			}
-		}
+		break
```

Comment:
The break should be within the if block.

Reply:
lol I have to run and did it in a hurry without looking. I'll fix it when I'm back in an hour.

Reply:
Fixed.